www/app/create/create.html

```diff
@@ -1,16 +1,17 @@
 <ion-view>
   <ion-content class="padding text-center">
   <!-- Text after angular text is placeholder -->
-    <h2>{{vm.name}} The Irish Bank Bar & Restaurant</h2>
-    <p>{{vm.types}} Irish, Irish Pub</p>
-    <p>{{vm.address}} 10 Mark Ln</p>
+    <h2>{{vm.name}}</h2>
+    <div ng-repeat="line in vm.address">
+      {{line}}
```

Comment:
The way the address is pulled from the database may need to be updated based on the new data model. See Kathy's PR for grabbing the data from the db and for the html to display the address.